scripts/gen_checkerboard.py

```diff
@@ -83,7 +83,14 @@ def make_gradient(n_inlines, n_crosslines, n_depth, box_size, dir="inline"):
     :return: numpy array
     """
 
-    axis = GRADIENT_DIR.index(dir)
+    _dir = dir # for depth case
+    if dir=='inline':
+        _dir = 'crossline'
+    elif dir=='crossline':
+        _dir = 'inline'
+    
+    axis = GRADIENT_DIR.index(_dir)
```

Comment:
I know we just discussed this issue, but it might be helpful to use a more descriptive variable name instead of `_dir` to help users better understand its role. Maybe something like `orthogonal_dir` ?